go.mod

```diff
@@ -4,7 +4,7 @@ go 1.12
 
 require (
 	github.com/blang/semver v3.6.1+incompatible
-	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200129194125-99a82ef07ec0
-	github.com/pkg/errors v0.8.1
+	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200427133649-8f7f388eb27f
```

Comment:
I am confused with the version requirement here, shouldn't a recent version be required here for Autocomplete?

Reply:
@levb, now it points to the feature branch of a server. When we merge branch to the master I'll change this to point on the recent version.